src/WalkingModule/src/Module.cpp

```diff
@@ -770,6 +784,7 @@ bool WalkingModule::updateModule()
                 }
                 m_newTrajectoryRequired = false;
                 resetTrajectory = true;
+                m_newTrajectoryMerged = true;
```

Comment:
Is this needed? If not, please, remove also its declaration in the class.

Reply:
Leftover, removed thanks